stan/math/prim/mat/prob/bernoulli_logit_glm_lpmf.hpp

```diff
@@ -107,7 +110,7 @@ typename return_type<T_x, T_alpha, T_beta>::type bernoulli_logit_glm_lpmf(
   T_partials_return theta_derivative_sum = 0;
```

Comment:
Is there a reason we need `theta_derivative`? It seems like we could increment `theta_derivative_sum` directly.